doc/classes/CanvasItem.xml

```diff
@@ -100,6 +100,7 @@
 			<argument index="3" name="antialiased" type="bool" default="false">
 			</argument>
 			<description>
+				Draw a multiline with a uniform [code]color[/code] and [code]width[/code] and optional antialiasing.
```

Comment:
* Draw -> Draws
* "multiline" is in the name, but may not be understood. Maybe "Draws a line connecting a series of points"?